### PR TITLE
Adds the Cleaning Smoke stage 3 syndrome.

### PR DESCRIPTION
**Pull request by Irradiation**
The counterpart to Colorsmoke, I guess.

:cl:
 * rscadd: Adds the Cleaning Smoke stage 3 virus syndrome. The Janitor will love it.

### CODE REVIEW

File: code/modules/virus2/effect/stage_3.dm [hunk 1]
```diff
@@ -520,6 +520,17 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/mob)
...
+/datum/disease2/effect/cleansmoke
+	name = "Cleaning Syndrome"
+	desc = "Causes the infected to synthesize smoke & space cleaner."
+	stage = 3
+	badness = EFFECT_DANGER_HELPFUL
+
+/datum/disease2/effect/cleansmoke/activate(var/mob/living/mob)
+	to_chat(mob, "<span class='notice'>You feel clean!</span>")
+	mob.reagents.add_reagent(CLEANER, 5)
+	mob.reagents.add_reagent(PAISMOKE, 5)
```
Comment (jknpj) [inline code/modules/virus2/effect/stage_3.dm:532]: I hate every single thing that uses PAISMOKE that's not a pAI

Comment (Irradiation) [reply code/modules/virus2/effect/stage_3.dm]: And I hate spawning the 3 chems that compose smoke and having them react with something else before poofing into smoke.

Comment (jknpj) [reply code/modules/virus2/effect/stage_3.dm:532]: Keep in mind that PAISMOKE also cleans the mob's reagents, not just makes smoke.

If you have making smoke manually, add a SMOKE reagent.

Comment (Exxion) [reply code/modules/virus2/effect/stage_3.dm:532]: Smoke always clears reagents
That's how it works

**Comment by Kurfursten** [comment]
It's fine to keep using PAISMOKE for this, I came to peace with it after Shadowmech fucking gave it to borers that powercreper

**Pull request merged**